cime_config/machines/cmake_macros/intel.cmake

```diff
@@ -0,0 +1,48 @@
+set(CFLAGS "-O2 -fp-model precise -std=gnu99")
+if (compile_threaded)
+  string(APPEND CFLAGS " -qopenmp")
+endif()
+if (NOT DEBUG)
+  string(APPEND CFLAGS " -O2 -debug minimal")
+endif()
+if (DEBUG)
```

Comment:
can this if (DEBUG) block be combined with the one at line 15?  I realize this is probably the raw output from your conversion tool.

Reply:
yeah, this is from the conversion tool. I could try to change it to group items with the same selectors, but that would change flag order which could change behavior.

Reply:
Actually I think the grouping might be fine as is as sections for different compilers (C, C++, Fortran).  It would help to have some comment strings to clarify that.